src/app/server/BUILD.gn

```diff
@@ -62,6 +62,11 @@ static_library("server") {
     "${chip_root}/src/transport",
   ]
 
+  # TODO: Server.cpp uses TestGroupData.h. Unsure why test code would be in such a central place
```

Comment:
Should we have an issue to track this?

Reply:
added #31831 . I am not familiar with groups so unsure how to quickly fix this (if applicable).